### PR TITLE
2021.3

### PR DESCRIPTION
**Pull request by ice1000**
This PR is created so it's easy to see my effort of making IntelliJ-Arend compatible with 2021.3 (currently EAP)

### CODE REVIEW

**Comment by ice1000** [comment]
[Arend-1.7.0.zip](https://github.com/JetBrains/intellij-arend/files/7240137/Arend-1.7.0.zip)
This is what I had for now... Managed to compile the project.

File: build.gradle.kts [hunk 1]
```diff
@@ -60,6 +60,7 @@ intellij {
     updateSinceUntilBuild.set(true)
     instrumentCode.set(true)
     plugins.set(listOf("yaml", "java", "IdeaVIM:0.69"))
+    localPath.set(property("localPath").toString())
```
Comment (valis) [inline build.gradle.kts:63]: What's that?

Comment (ice1000) [reply build.gradle.kts:63]: This is a mistake, good catch. I'm deleting it

**Comment by ice1000** [comment]
[Arend-1.7.0.zip](https://github.com/JetBrains/intellij-arend/files/7243556/Arend-1.7.0.zip)

An update

**Comment by valis** [comment]
Already merged

**Pull request closed without merging**